### PR TITLE
Some minor improvement to test messages

### PR DESCRIPTION
**Pull request by Geod24**
Was trying to make sense of some error and ended up with a few commits.
Not happy with how the whole thing look like but that's a battle for another day.

### CODE REVIEW

File: source/agora/test/Base.d [hunk 1]
```diff
@@ -788,14 +789,16 @@ public class TestAPIManager
...
-                    retryFor(this.clients[idx].getPreimages(Set!Hash.from(enroll.utxo_key.only))
+                {
+                    scope node = this.nodes[idx];
+                    retryFor(node.client.getPreimages(Set!Hash.from(enroll.utxo_key.only))
                         .any!(preimage => preimage.height >= height),
-                            timeout, format!"Client #%s has no preimage for client #%s at distance %s"
-                            (idx, idx_enroll, height));
-            }));
+                            timeout, format!"Client #%s (%s) has no preimage for client #%s (%s) at height %s"
+                             (idx, node.address, idx_enroll, this.nodes[idx_enroll].address, height));
```
Comment (linked0) [inline source/agora/test/Base.d]:
```suggestion
                                (idx, node.address, idx_enroll, this.nodes[idx_enroll].address, height));
```

File: source/agora/test/Base.d [hunk 2]
```diff
@@ -1223,11 +1226,12 @@ public class TestAPIManager
...
         auto enroll = clients[client_idx].setRecurringEnrollment(true);
-        client_idxs.each!(idx =>
+        foreach (idx; client_idxs)
             retryFor(this.clients[idx].getEnrollment(enroll.utxo_key) == enroll,
                 5.seconds,
-                format!"[%s:%s] Client #%s enrollment not in pool of client #%s"
-                    (file, line, client_idx, idx)));
+                format!"Client #%s (%s) enrollment not in pool of client #%s (%s)"
+                     (client_idx, this.nodes[client_idx].address, idx, this.nodes[idx].address),
+                     file, line);
```
Comment (linked0) [inline source/agora/test/Base.d]:
```suggestion
                 file, line);
```

**Review by linked0 — APPROVED** [review]
LGTM

**Comment by Geod24** [comment]
Review addressed.

**Pull request merged**